controller/src/beerocks/master/son_master_thread.cpp

```diff
@@ -818,6 +818,13 @@ bool master_thread::handle_cmdu_1905_autoconfiguration_WSC(const std::string &sr
     }
 
     if (!database.setting_certification_mode()) {
+        // trigger Topology query
```

Comment:
Commit message typo - "upn" --> "upon"

Also, the commit message doesn't explain at all why we send the topology query at this time.
Finally - why do we send it only in certification mode? This will mean it will not work in RDKB and UGW since by default certification mode is not set in these platforms.

Reply:
done